forwarder/parse_config_update.py

```diff
@@ -37,12 +37,14 @@ class ConfigUpdate:
 
 
 def parse_config_update(config_update_payload: str) -> Optional[ConfigUpdate]:
-    config = json.loads(config_update_payload)
     try:
+        config = json.loads(config_update_payload)
         command_type = CommandType(config["cmd"])
     except KeyError:
         logger.warning('Message received in config topic contained no "cmd" field')
         return None
+    except json.JSONDecodeError:
+        logger.warning(f'Invalid command received')
```

Comment:
This change is a little more clear about what went wrong and also makes `black` happy with the formatting:
```suggestion
        logger.warning(f"Command received was not recognised as valid JSON")
```

Reply:
I have to use 
```python
logger.warning("Command received was not recognised as valid JSON")
```
flake8 was not happy either: ` f-string is missing placeholders`